netutils/iperf/iperf.c

```diff
@@ -196,6 +204,47 @@ static int iperf_show_socket_error_reason(FAR const char *str, int sockfd)
   return err;
 }
 
+/****************************************************************************
+ * Name: iperf_print_addr
+ *
+ * Description:
+ *   Print addr info
+ *
+ ****************************************************************************/
+
+static void iperf_print_addr(FAR const char *str, FAR struct sockaddr *addr)
+{
+  switch (addr->sa_family)
+    {
+      case AF_INET:
+        {
+          FAR struct sockaddr_in *inaddr = (FAR struct sockaddr_in *)addr;
+          printf("%s: %s:%d\n", str, inet_ntoa(inaddr->sin_addr),
+                                         htons(inaddr->sin_port));
```

Comment:
```suggestion
          printf("%s: %s:%d\n", str, inet_ntoa(inaddr->sin_addr),
                                               htons(inaddr->sin_port));
```

Reply:
Then maybe I can align `htons`'s 'h' with `inet_ntoa`'s 'i' (like newest patch)

Reply:
Align with `"%s: %s:%d\n"` as I made a mistake, by assuming that `htons` was passed as a second parameter to `inet_ntoa `

Reply:
Done.